### DISCUSSION TITLE
How to contribute to adding more colour choices?

### DISCUSSION THREAD

**Discussion by ghost** [squidfunk/mkdocs-material#5439] [Q&A] [answered]
I want more colour choices for which I have HEX code but no human readable name. It does respect material design colour palette patterns or approach. So, how to contribute to adding more colour choices to this project?

Do you think this could be good candidate to convert to issue and worked upon?

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking. We have integrated the most fitting tints of the Material Design palette, and we stick to that. If we would add more color choices, it would increase the size of the bundle, specifically [`palette.css`](https://github.com/squidfunk/mkdocs-material/blob/4006b4f04d83ebb20ff095d39457a9a17ad33cae/material/assets/stylesheets/palette.a6bdf11c.min.css), and thus negatively impact performance. When making choices what to integrate, we always have to strike for a balance between convenience, utility and performance. For colors, this choice has proven to be optimal. Additionally, we offer ways to override the defaults with your custom colors, [see our documentation](https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#custom-colors).

Good defaults and easy extensibility – that's what we aim for with everything we do.

**Reply by ghost** [reply]
Thanks for the quick response, @squidfunk. Custom colours may work for me. Will try them today. Thanks once again.
